Assets/MixedRealityToolkit/Utilities/Physics/TransformHelper.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Physics
+{
+    /// <summary>
+    /// Helper class containing move/rotate/scale related utility functions.
```

Comment:
I don't like that we're storing those min max values in two different classes. Would it be an option to remove them here and instead change SetScaleLimits to AdjustScaleLimits(..) which does the initialscale multiplication but returns that value instead of storing it and ClampScale would then take the max scale values as second parameter ? that way you keep the data in one place and it doesn't diverge

Reply:
Ah, there is one issue worth discussing here. If we do things like this, if you have both a `ManipulationHandler` and a `BoundingBox` on an object, the object will have two different min/max scales.

Say for example someone had MH max scale as 3 and BB max scale as 5. If they scaled to max using the BB scale handles and used MH two hand manipulation, the scale would jump down to three.

Reply:
Previously there was conflict as to which min/max scale would be taken, but at least there was one authoritative min/max scale.